### ISSUE TITLE
Tag name should be alphanumeric error message triggered when wrong flag is used

### ISSUE THREAD

**Issue by jingxueguo** [jingxueguo/ped#6]
![image.png](https://raw.githubusercontent.com/jingxueguo/ped/main/files/e60aa679-11a9-46d6-8e38-e0a8bf82a92b.png)

Had a typo during testing, used "-tt" instead of "-t" for the tag. Expected error message but it seems that this is still recognised as a tag flag.

In addition, when tested with a dummy flag that started with d, it was also recognised as data flag. It could be better if the app is more precise as to what is considered a flag and throw an error if wrong flag is used, rather than using the first letter.

In short, error message probably should state "wrong flag used" rather than the data or tab being in the wrong format

![image.png](https://raw.githubusercontent.com/jingxueguo/ped/main/files/8d842984-f8db-41c9-b793-97a7ac18667e.png)

![image.png](https://raw.githubusercontent.com/jingxueguo/ped/main/files/fb054117-a1e5-4bca-80e7-0238e1a289e2.png)
